app/assets/stylesheets/modern-business.scss

```diff
@@ -273,6 +282,9 @@ footer {
   text-align: center;
 }
 
+#undo-link {
+  color: #E74C3C;
```

Comment:
Color literals like `#E74C3C` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E74C3C` should be written as `#e74c3c`